### DISCUSSION TITLE
Style Guide, Issue Templates and Contributing Guide

### DISCUSSION THREAD

**Discussion by gamemaker1** [nfriedly/express-rate-limit#375] [Ideas]
Since all of these are the same across all the repositories anyways, why not extract them into separate packages? It'd be much easier to maintain.

The typescript configuration could be uploaded to npm as a new package called `@express-rate-limit/tsconfig`, prettier configuration could be `@express-rate-limit/prettier`, and the issue templates/pr templates/contributing guide could be put into the special `.github` repo (see [1](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) and [2](https://github.com/orgs/community/discussions/22451)).

**Comment by nfriedly** [comment]
Yeah, that seems like a good idea.

**Reply by gamemaker1** [reply]
Created [`@express-rate-limit/tsconfig`](https://www.npmjs.com/package/@express-rate-limit/tsconfig) and [`@express-rate-limit/prettier`](https://www.npmjs.com/package/@express-rate-limit/prettier), and invited you to the org on npm.
